pod/perlbook.pod

```diff
@@ -94,7 +94,7 @@ You might want to keep these desktop references close by your keyboard:
  by Richard Foley
  ISBN 978-0-596-00503-0 [1st edition January 2004]
  ISBN 978-0-596-55625-9 [ebook]
- https://oreilly.com/catalog/9780596005030/
+ https://www.oreilly.com/library/view/-/9781449311186/
```

Comment:
This is weird (because I already changed it):

https://github.com/Perl/perl5/pull/22186/files#diff-6483ac54a634441f897d3d018524d26b358df2432f96ca51b0672758da6ef994R104